project/Deliverable0.md

```diff
@@ -80,20 +80,7 @@ In the meantime for the first week, you should go to a lab even if you are not r
 
 **Q. How should I format my tests?**
 
-A. You can use the async/await syntax like in the bootstrap,
-```typescript
-it("Should do something", async () => {
-    let response: type;
-    try {
-        response = await insightFacade.methodName(params);
-    } catch (err) {
-        response = err;
-    } finally {
-        expect(response).to.deep.equal(expectedValue);
-    }
-});
-```
-or you can return the promise created by your function call, and use appropriate expect statements in the `then()` and `catch()` blocks. For example:
+You can return the promise created by your function call, and use appropriate expect statements in the `then()` and `catch()` blocks. For example:
```

Comment:
Can we start this line with `A. `? Or replace the example with with a then/catch formatted one?

Reply:
You got it champ